packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -40,13 +40,13 @@ export const SimpleDocument: FC<Props> = ({
   let html,
     passage: QueryResultPassage | null = null;
   if (document) {
-    //Json object usually don't have enough info to allow us to determine which field to display
-    //Unless there is passage pointing to a specific field.
+    // JSON objects will default to displaying the specfied body field, text field, or passage highlighting field,
+    // Otherwise an error is shown
     const isJsonType = get(document, 'extracted_metadata.file_type') === 'json';
```

Comment:
Use refactored document utils function (same as used in `DocumentPreview.tsx` above).